index.html

```diff
@@ -425,8 +423,12 @@ <h2>kid</h2>
         <p>
       If <code>kid</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>,
       a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
-      to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
-      <a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
+      as a hint indicating which key was used to secure the verifiable credential when performing a
+      <a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process as defined in <a data-cite="RFC7515#section-4.1.4">RFC7515</a>.
+        </p>
+        <p>
+          <code>kid</code> MUST be present when the key of the <a data-cite="VC-DATA-MODEL#dfn-issuers">issuer</a>
+          or <a data-cite="VC-DATA-MODEL#dfn-subjects">subject</a> is expressed as a DID.
```

Comment:
We need to specify the exact format that we expect here... DID URL -- but is this a relative (frament-only thing) or a fully qualified DID URL (DID URL + fragment). What issue is dealing w/ this?

Reply:
@msporny I added a link to your comment here: https://github.com/w3c/vc-jose-cose/issues/117#issuecomment-1726208766

To hold the issue open until its addressed.